kobo/apps/stripe/constants.py

```diff
@@ -27,10 +27,3 @@
     'storage': 'storage_bytes',
     'submission': 'submission',
 }
-
-USAGE_LIMIT_MAP_STRIPE = {
```

Comment:
If I'm not mistaken, this alternative mapping const is no longer needed. We use asr/mt rather than 'nlp' in stripe products now, right?